sycl/CMakeLists.txt

```diff
@@ -188,10 +188,9 @@ set(SYCL_EXT_ONEAPI_BACKEND_CUDA ${LLVM_HAS_NVPTX_TARGET})
 set(SYCL_EXT_ONEAPI_BACKEND_HIP ${LLVM_HAS_AMDGPU_TARGET})
 
 # Configure SYCL version macro
-set(sycl_inc_dir ${CMAKE_CURRENT_SOURCE_DIR}/include)
+set(SYCL_ROOT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(sycl_src_dir ${CMAKE_CURRENT_SOURCE_DIR}/source)
```

Comment:
```suggestion
set(sycl_src_dir ${SYCL_ROOT_SOURCE_DIR}/source)
```

Reply:
Applied in 7123b46c9ba4aad9ad174a2959b181e8c9f765d1